runtime/manual/basics/modules/index.md

```diff
@@ -1,237 +0,0 @@
-contains `main.ts` and `calc.ts`.
-
-## Adding third party modules and libraries to deno.json
-
-Using remote modules (downloaded from a registry) in Deno uses the same `import`
-syntax as your local code.
-
-The modules that you add to your project are tracked as `imports` in
-`deno.json` - we call this the import map.
-
-```json title="deno.json"
-{
-  "imports": {
-    "@scopename/mypackage": "jsr:@scopename/mypackage@^16.1.0"
-  }
-}
-```
-
-You can add modules using the `deno add` subcommand:
-
-```sh
-# Add the latest version of the module to deno.json
-$ deno add @luca/cases
-Add @luca/cases - jsr:@luca/cases@^1.0.0
-```
-
-```json title="deno.json"
-{
-  "imports": {
-    "@luca/cases": "jsr:@luca/cases@^1.0.0"
-  }
-}
-```
-
-The `deno add` command will automatically add the latest version of the module
-you requested to your project imports, unless you specify an exact version:
-
-```sh
-# Passing an exact version
-$ deno add @luca/cases@1.0.0
-Add @luca/cases - jsr:@luca/cases@^1.0.0
-```
-
-## Using installed modules
-
-Once a package is added to the import map in `deno.json`, you can import the
-module by its name, and Deno will automatically resolve the module. For example:
-
-```ts title="main.ts"
-import { camelCase } from "@luca/cases";
-
-camelCase("hello world"); // "helloWorld"
-```
-
-## Package Registries
-
-Deno recommends that you use [JSR](https://jsr.io/), the JavaScript registry, to
-publish and manage your modules for the best publishing experience. JSR offers
-automatic documentation generation, semver resolution and improved performance
-for TypeScript code. Deno also supports
-[other platforms](https://jsr.io/docs/other-registries) for publishing modules,
-such as npm, and JavaScript CDNs like deno.land/x, esm.h and unpkg.com
-
-By default when you use the CLI, the package will be resolved from JSR.
-
-```bash
-deno add @scopename/mypackage
-```
-
-The resulting import map contains the `jsr:` import specifier in the resulting
-`deno.json` file.
-
-```json
-{
-  "imports": {
-    "@scopename/mypackage": "jsr:@scopename/mypackage@^16.1.0"
-  }
-}
-```
-
-## Importing from other package registries
-
-Using existing packages from registries like `npm` is equivalent to the
-experience in `node`. To add a package from `npm` you need to prefix the package
-name with the `npm:` specifier:
-
-```bash
-$ deno add npm:uuid
-Add uuid - npm:uuid@^10.0.0
-```
-
-The resulting `deno.json` will look like this:
-
-```json
-{
-  "imports": {
-    "uuid": "npm:uuid@^10.0.0"
-  }
-}
-```
-
-You will then be able to use the `npm` module by importing it with the name from
-the import map.
-
-```ts title="mod.ts"
-import * as uuid from "uuid";
-
-console.log(uuid.v4());
-```
-
-### Supported package registries and specifiers
-
-We support the following registries:
-
-| Registry | Specifier |
-| -------- | --------- |
-| JSR      | jsr:      |
-| npm      | npm:      |
```

Comment:
This information is important and seems excluded

Reply:
its in there, line 102/